### PR TITLE
CI: Add 3.7 and nightly builds.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

**Comment by mrakitin** [comment]
Oh, got it. I think the Ubuntu version is outdated there (v14), that's why it cannot find the Python distribution. I think we saw it with https://github.com/bluesky/bluesky/pull/1203.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 language: python
```
Comment (mrakitin) [inline .travis.yml]:
```suggestion
dist: xenial
language: python
```

Comment (danielballan) [reply .travis.yml]: Thanks, I would have sworn I included that. Not sure how I missed it!

Comment (mrakitin) [reply .travis.yml]: Actually it should be enabled by default. Probably it wasn't picket up due to some weird caching on the TravisCI side.

**Pull request merged**